src/analyzer.ts

```diff
@@ -109,12 +110,23 @@ export class Analyzer {
         document: Document<any, any>): Promise<DocumentDescriptor> {
     let entities = await this.getEntities(document);
 
-    // TODO(justinfagnani): Load ImportDescriptors
-
-    return new DocumentDescriptor(document, entities);
+    let dependencyDescriptors: Descriptor[] = entities
+        .filter((e) => e instanceof InlineDocumentDescriptor
+          || e instanceof ImportDescriptor);
+    let analyzeDependencies = dependencyDescriptors
+        .map((d) =>  {
+          if (d instanceof InlineDocumentDescriptor) {
+            return this.analyzeSource(d.type, d.contents, document.url);
+          } else if (d instanceof ImportDescriptor) {
+            return this.analyze(d.url);
```

Comment:
The resolution is done in the entity finder... I suppose there's a question about where to do it, but I added to the test to make sure it's done.